data_steward/analytics/tools/email_generator/dictionaries_and_lists.py

```diff
@@ -0,0 +1,335 @@
+    'Date/Datetime Disparity': 'maximum',
+
+    # other metrics
+    'Concept ID Success Rate': 'minimum',
+    'Duplicate Records': 'maximum',
+    'Erroneous Dates': 'maximum',
+    'Person ID Failure Rate': 'maximum'}
+
+percentage_dict = {
+    'duplicates': False,
+    'data_after_death': True,
+    'end_before_begin': True,
+    'concept': True,
+    'measurement_units': True,
+    'drug_routes': True,
+    'drug_success': True,
+    'sites_measurement': True,
+    'visit_date_disparity': True,
+    'date_datetime_disparity': True,
+    'erroneous_dates': True,
+    'person_id_failure_rate': True}
+
+target_low_dict = {
+    'duplicates': True,
+    'data_after_death': True,
+    'end_before_begin': True,
+    'concept': False,
+    'measurement_units': False,
+    'drug_routes': False,
+    'drug_success': False,
+    'sites_measurement': False,
+    'visit_date_disparity': False,
+
+    # FIXME: the three below - by logic - should
+    # be 'True' but were calculated as showing
+    # the % of errors rather than (100 - % of errors)
+    # in the DQM scripts. This means they should be
+    # logged as 'False' here to make it an effective
+    # double negative.
+    'date_datetime_disparity': False,
+    'erroneous_dates': False,
+    'person_id_failure_rate': False}
+
+
+# NOTE: This is actually different than what one would
+# find in the metrics_over_time generator. This is because
+# we want less granularity for the 'integration metrics'.
+
+columns_to_document_for_sheet_email = {
+    'measurement_units': ['total_unit_success_rate'],
+
+    'sites_measurement': [
+        'Physical_Measurement',	'CMP', ' CBCwDiff',
+        'CBC', 'Lipid',	'All_Measurements'],
+
+    'end_before_begin': [
+        'visit_occurrence', 'condition_occurrence',
+        'drug_exposure', 'device_exposure'],
+
+    'duplicates': ['visit_occurrence', 'condition_occurrence',
+                   'drug_exposure', 'measurement',
+                   'procedure_occurrence', 'device_exposure',
+                   'observation'],
+
+    'drug_routes': ['total_route_success_rate'],
+
+    'drug_success': [
+        'ace_inhibitors', 'painnsaids', 'msknsaids',
+        'statins', 'antibiotics', 'opioids', 'oralhypoglycemics',
+        'vaccine', 'ccb', 'diuretics', 'all_drugs'],
+
+    'data_after_death': [
+        'visit_occurrence', 'condition_occurrence',
+        'drug_exposure', 'measurement',
+        'procedure_occurrence', 'observation'],
+
+    'diabetes': [
+        'diabetics_w_drugs', 'diabetics_w_glucose',
+        'diabetics_w_a1c', 'diabetics_w_insulin'],
+
+    'concept': [
+        'observation_success_rate', 'drug_success_rate',
+        'procedure_success_rate', 'condition_success_rate',
+        'measurement_success_rate', 'visit_success_rate'],
+
+    'date_datetime_disparity': [
+        'visit_occurrence', 'condition_occurrence',
+        'drug_exposure', 'measurement',
+        'procedure_occurrence', 'observation'],
```

Comment:
Are you considering adding all the constants in a new file? A meta-python-script could parse these python files for strings and do the job of creating and replacing the constants for you.